docs/site/Fields-filter.md

```diff
@@ -0,0 +1,86 @@
+---
+title: 'Fields filter'
+lang: en
+keywords: LoopBack 4.0, LoopBack 4, fields
+layout: readme
+source: loopback-next
+sidebar: lb4_sidebar
+permalink: /doc/en/lb4/Fields-filter.html
+---
+
+A _fields_ filter specifies properties (fields) to include or exclude from the results.
+
+### REST API
+
+
+**The <true|false> does not work for url as the boolean is being concerted to string, which is always trru**
```

Comment:
typo: `concerted` -> `converted`
typo: `trru` -> `true`

Also this sentence confuses me until I see the example below, can you clarify what you mean by `work for url`?

Reply:
This filter is buggy. Try to find a way to explain it. Let me reword,